manifests/l/LightBurnSoftware/LightBurn/1.2.01/LightBurnSoftware.LightBurn.installer.yaml

```
@@ -0,0 +1,47 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=NVS2.5-1-22610-1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: LightBurnSoftware.LightBurn
+PackageVersion: 1.2.01
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /verysilent
+  SilentWithProgress: /silent
+UpgradeBehavior: install
+Protocols:
+- lbrn
+- lbrn2
+FileExtensions:
+- ai
+- dxf
+- gif
+- hpgl
+- jpeg
+- jpg
+- lbrn
+- lbrn2
+- pdf
+- plt
+- png
+- rd
+- sc
+- scpro2
+- svg
+- tga
+- tif
+ReleaseDate: 2022-07-21
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/lightburnsoftware/deployment/releases/download/1.2.01/lightburn-v1.2.01.exe
+  InstallerSha256: CBFFB6E0177EB72BA888AEEBA3FC666C6AEE5AF5748BB2BB01B732599DE972B3
+- Architecture: x86
+  InstallerUrl: https://github.com/lightburnsoftware/deployment/releases/download/1.2.01/lightburn-32Bit-v1.2.01.exe
```

```suggestion
  InstallerUrl: https://github.com/LightBurnSoftware/deployment/releases/download/1.2.01/LightBurn-32Bit-v1.2.01.exe
```
Keep the capitalization as is here since changing the capitalization will cause a 301 redirect from lowercase to upper-case and then the final redirect.